e2e/helpers/helpers.ts

```diff
@@ -92,8 +94,18 @@ export class HelpFunctions {
     await this.sessionAcceptButton.click();
   }
 
-  async assertCallActive() {
-    await expect(this.sessionStatus).toHaveText('active');
+  async assertCallStatus(status: string) {
```

Comment:
A nice little trick you can do in Typescript to make sure you don't have typo's and such:

```suggestion
  async assertCallStatus(status: 'ringing' | 'active' | 'on_hold') {
```

Then for instance calling `assertCallStatus('lol')` will result in an error in your editor. And it will also give you auto completion in your editor :D